src/drivers/main.cpp

```diff
@@ -1,59 +1,47 @@
+#include <cassert>
 #include <iostream>
+#include <precice/SolverInterface.hpp>
+#include <precice/Tooling.hpp>
+#include <stdexcept>
 #include <string>
-#include "precice/config/Configuration.hpp"
-#include "precice/impl/versions.hpp"
-#include "utils/assertion.hpp"
-#include "xml/Printer.hpp"
 
 void printUsage()
 {
-  std::cout << "Usage:\n\n";
-  std::cout << "Print XML reference      :  binprecice xml\n";
-  std::cout << "Print DTD for XML config :  binprecice dtd" << std::endl;
-  std::cout << "Print Markdown reference :  binprecice md" << std::endl;
+  std::cerr << "Usage:\n\n";
+  std::cerr << "Print XML reference      :  binprecice xml\n";
+  std::cerr << "Print DTD for XML config :  binprecice dtd\n";
+  std::cerr << "Print Markdown reference :  binprecice md\n";
+  std::cerr << "Print preCICE version    :  binprecice version\n";
+  std::cerr << "Check configuration      :  binprecice check FILE [ PARTICIPANT [ COMMSIZE ] ]\n";
```

Comment:
`binprecice check` is not mentioned in the changelog.

Reply:
This somehow sneaked in from another branch. Thanks for pointing this out!